[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[2] libensemble.libE_worker (INFO): No job_controller set on worker 2
[3] libensemble.libE_worker (INFO): No job_controller set on worker 3
[4] libensemble.libE_worker (INFO): No job_controller set on worker 4
[2] libensemble.libE_worker (INFO): Worker 2 initiated on node mcswl054.mcs.anl.gov
[3] libensemble.libE_worker (INFO): Worker 3 initiated on node mcswl054.mcs.anl.gov
[4] libensemble.libE_worker (INFO): Worker 4 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE (INFO): libE_manager total time: 1.0030567646026611
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: name 'b' is not defined
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 30, in mop_mod_wrapper
    O = np.zeros(b, dtype=gen_specs['out'])
NameError: name 'b' is not defined

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[2] libensemble.libE_worker (INFO): No job_controller set on worker 2
[3] libensemble.libE_worker (INFO): No job_controller set on worker 3
[4] libensemble.libE_worker (INFO): No job_controller set on worker 4
[2] libensemble.libE_worker (INFO): Worker 2 initiated on node mcswl054.mcs.anl.gov
[3] libensemble.libE_worker (INFO): Worker 3 initiated on node mcswl054.mcs.anl.gov
[4] libensemble.libE_worker (INFO): Worker 4 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE (INFO): libE_manager total time: 0.9643499851226807
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[2] libensemble.libE_worker (INFO): No job_controller set on worker 2
[3] libensemble.libE_worker (INFO): No job_controller set on worker 3
[4] libensemble.libE_worker (INFO): No job_controller set on worker 4
[2] libensemble.libE_worker (INFO): Worker 2 initiated on node mcswl054.mcs.anl.gov
[3] libensemble.libE_worker (INFO): Worker 3 initiated on node mcswl054.mcs.anl.gov
[4] libensemble.libE_worker (INFO): Worker 4 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE (INFO): libE_manager total time: 0.9819581508636475
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[2] libensemble.libE_worker (INFO): No job_controller set on worker 2
[3] libensemble.libE_worker (INFO): No job_controller set on worker 3
[4] libensemble.libE_worker (INFO): No job_controller set on worker 4
[2] libensemble.libE_worker (INFO): Worker 2 initiated on node mcswl054.mcs.anl.gov
[3] libensemble.libE_worker (INFO): Worker 3 initiated on node mcswl054.mcs.anl.gov
[4] libensemble.libE_worker (INFO): Worker 4 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE (INFO): libE_manager total time: 0.9689521789550781
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[2] libensemble.libE_worker (INFO): No job_controller set on worker 2
[3] libensemble.libE_worker (INFO): No job_controller set on worker 3
[4] libensemble.libE_worker (INFO): No job_controller set on worker 4
[2] libensemble.libE_worker (INFO): Worker 2 initiated on node mcswl054.mcs.anl.gov
[3] libensemble.libE_worker (INFO): Worker 3 initiated on node mcswl054.mcs.anl.gov
[4] libensemble.libE_worker (INFO): Worker 4 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE (INFO): libE_manager total time: 0.9955019950866699
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[2] libensemble.libE_worker (INFO): No job_controller set on worker 2
[3] libensemble.libE_worker (INFO): No job_controller set on worker 3
[4] libensemble.libE_worker (INFO): No job_controller set on worker 4
[2] libensemble.libE_worker (INFO): Worker 2 initiated on node mcswl054.mcs.anl.gov
[3] libensemble.libE_worker (INFO): Worker 3 initiated on node mcswl054.mcs.anl.gov
[4] libensemble.libE_worker (INFO): Worker 4 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE (INFO): libE_manager total time: 0.986685037612915
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[2] libensemble.libE_worker (INFO): No job_controller set on worker 2
[3] libensemble.libE_worker (INFO): No job_controller set on worker 3
[4] libensemble.libE_worker (INFO): No job_controller set on worker 4
[2] libensemble.libE_worker (INFO): Worker 2 initiated on node mcswl054.mcs.anl.gov
[3] libensemble.libE_worker (INFO): Worker 3 initiated on node mcswl054.mcs.anl.gov
[4] libensemble.libE_worker (INFO): Worker 4 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE (INFO): libE_manager total time: 0.9992308616638184
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: write_record() got an unexpected keyword argument 'dtype'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 28, in mop_mod_wrapper
    fp1.write_record(d, dtype=np.int32)
TypeError: write_record() got an unexpected keyword argument 'dtype'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 50, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 50, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 50, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 50, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: name 'dp' is not defined
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 31, in mop_mod_wrapper
    fp1.write_record(np.array(lb, dtype=dp))
NameError: name 'dp' is not defined

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: name 'dp' is not defined
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 31, in mop_mod_wrapper
    fp1.write_record(np.array(lb, dtype=dp))
NameError: name 'dp' is not defined

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 50, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 49, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 48, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: [Errno 2] No such file or directory: 'mop.out'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 48, in mop_mod_wrapper
    fp2 = FortranFile('mop.out','r')
  File "/Users/tylerchang/Library/Python/3.7/lib/python/site-packages/scipy/io/_fortran.py", line 104, in __init__
    self._fp = open(filename, '%sb' % mode)
FileNotFoundError: [Errno 2] No such file or directory: 'mop.out'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: only integer scalar arrays can be converted to a scalar index
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 51, in mop_mod_wrapper
    cand_pts.reshape(nb,d) # reshape to appropriate dimensions
TypeError: only integer scalar arrays can be converted to a scalar index

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: only integer scalar arrays can be converted to a scalar index
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 52, in mop_mod_wrapper
    cand_pts.reshape(nb,d) # reshape to appropriate dimensions
TypeError: only integer scalar arrays can be converted to a scalar index

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: only integer scalar arrays can be converted to a scalar index
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 52, in mop_mod_wrapper
    cand_pts.reshape(nb,d) # reshape to appropriate dimensions
TypeError: only integer scalar arrays can be converted to a scalar index

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: module 'numpy' has no attribute 'flat64'
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 50, in mop_mod_wrapper
    cand_pts = np.zeros(nb*d,dtype=np.flat64)
AttributeError: module 'numpy' has no attribute 'flat64'

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: only integer scalar arrays can be converted to a scalar index
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 52, in mop_mod_wrapper
    cand_pts.reshape(nb,d) # reshape to appropriate dimensions
TypeError: only integer scalar arrays can be converted to a scalar index

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: only integer scalar arrays can be converted to a scalar index
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 52, in mop_mod_wrapper
    cand_pts.reshape(nb,d) # reshape to appropriate dimensions
TypeError: only integer scalar arrays can be converted to a scalar index

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: only integer scalar arrays can be converted to a scalar index
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 53, in mop_mod_wrapper
    for i in range(1,nb):
TypeError: only integer scalar arrays can be converted to a scalar index

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: only integer scalar arrays can be converted to a scalar index
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 54, in mop_mod_wrapper
    for i in range(1,nb):
TypeError: only integer scalar arrays can be converted to a scalar index

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: could not broadcast input array from shape (31) into shape (30,2)
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 58, in mop_mod_wrapper
    O['x'] = x_vals
ValueError: could not broadcast input array from shape (31) into shape (30,2)

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: could not broadcast input array from shape (31) into shape (2,30,2)
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 58, in mop_mod_wrapper
    O['x'] = x_vals
ValueError: could not broadcast input array from shape (31) into shape (2,30,2)

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: could not broadcast input array from shape (31) into shape (2,30,2)
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 58, in mop_mod_wrapper
    O['x'] = x_vals
ValueError: could not broadcast input array from shape (31) into shape (2,30,2)

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: could not broadcast input array from shape (60) into shape (30,2)
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 58, in mop_mod_wrapper
    O['x'] = x_vals
ValueError: could not broadcast input array from shape (60) into shape (30,2)

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: name 'x_vals' is not defined
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 57, in mop_mod_wrapper
    O['x'] = x_vals
NameError: name 'x_vals' is not defined

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE (ERROR): ---- Received error message from 1 ----
[0] libensemble.libE (ERROR): Message: name 'x_vals' is not defined
[0] libensemble.libE (ERROR): Traceback (most recent call last):
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 313, in run
    response = self._handle(Work)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 284, in _handle
    calc_out, persis_info, calc_status = self._handle_calc(Work, calc_in)
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 224, in _handle_calc
    out = calc(calc_in, Work['persis_info'], Work['libE_info'])
  File "/Users/tylerchang/Git/libensemble/libensemble/libE_worker.py", line 167, in run_gen
    return gen_f(calc_in, persis_info, gen_specs, libE_info)
  File "/Users/tylerchang/Git/libensemble/libensemble/gen_funcs/mop_mod.py", line 57, in mop_mod_wrapper
    O['x'] = x_vals
NameError: name 'x_vals' is not defined

[0] libensemble.libE (ERROR): Manager exception raised .. aborting ensemble:
[0] libensemble.libE (ERROR): Dumping ensemble history with 0 sims evaluated:
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[2] libensemble.libE_worker (INFO): No job_controller set on worker 2
[3] libensemble.libE_worker (INFO): No job_controller set on worker 3
[4] libensemble.libE_worker (INFO): No job_controller set on worker 4
[2] libensemble.libE_worker (INFO): Worker 2 initiated on node mcswl054.mcs.anl.gov
[3] libensemble.libE_worker (INFO): Worker 3 initiated on node mcswl054.mcs.anl.gov
[4] libensemble.libE_worker (INFO): Worker 4 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE (INFO): libE_manager total time: 6.969158172607422
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[2] libensemble.libE_worker (INFO): No job_controller set on worker 2
[3] libensemble.libE_worker (INFO): No job_controller set on worker 3
[4] libensemble.libE_worker (INFO): No job_controller set on worker 4
[2] libensemble.libE_worker (INFO): Worker 2 initiated on node mcswl054.mcs.anl.gov
[3] libensemble.libE_worker (INFO): Worker 3 initiated on node mcswl054.mcs.anl.gov
[4] libensemble.libE_worker (INFO): Worker 4 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE (INFO): libE_manager total time: 2.6862778663635254
[0] libensemble.libE_manager (INFO): Manager initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Manager exit_criteria: {'sim_max': 1000, 'elapsed_wallclock_time': 300}
[1] libensemble.libE_worker (INFO): No job_controller set on worker 1
[1] libensemble.libE_worker (INFO): Worker 1 initiated on node mcswl054.mcs.anl.gov
[2] libensemble.libE_worker (INFO): No job_controller set on worker 2
[3] libensemble.libE_worker (INFO): No job_controller set on worker 3
[4] libensemble.libE_worker (INFO): No job_controller set on worker 4
[2] libensemble.libE_worker (INFO): Worker 2 initiated on node mcswl054.mcs.anl.gov
[3] libensemble.libE_worker (INFO): Worker 3 initiated on node mcswl054.mcs.anl.gov
[4] libensemble.libE_worker (INFO): Worker 4 initiated on node mcswl054.mcs.anl.gov
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE_manager (INFO): Term test tripped: sim_max
[0] libensemble.libE (INFO): libE_manager total time: 2.675287961959839
